### DISCUSSION TITLE
Include the Go To Column option in the preview data functionality (like the one exist in Power Query)

### DISCUSSION THREAD

**Discussion by AntonioPalomas** [TabularEditor/TabularEditor3#1209] [Ideas]
### Is your feature request related to a problem? Please describe

Include the Go To Column option in the preview data functionality (like the one exist in Power Query)

### Describe the solution you'd like

When a table has many columns, finding a specific column is difficult because you have to use the scroll bar. If the Go To Column functionality is introduced, like the one we have in Power Query to choose the column and position it, it will improve usage and productivity.

### Goals

1. Improve usage and productivity.
2.

**Comment by mlonsk** [comment]
Hi @AntonioPalomas 

Thank you for the suggestion, I can see how this would improve usability, and I will run the idea by our development team.

**Comment by ttroutwine** [comment]
This would be an incredible feature to have.

**Comment by mlonsk** [comment]
Hi 
Tabular Editor 3.27.0 brings a feature to Track the TOM explorer in the Data Preview view. That is, when you click a column in the TOM Explorer it will highlight the column in the Data Preview. Do let us know how it works for you.
